Houseclub/src/main/java/me/grishka/houseclub/fragments/ProfileFragment.java

```diff
@@ -160,15 +171,92 @@ public void onSuccess(GetProfile.Response result){
 						}else{
 							inviterPhoto.setVisibility(View.GONE);
 						}
+
+						if(user.mutualFollows.size() > 0){
+
+							followed_by.setText(" Followed by ");
```

Comment:
is this related to upcoming events?

Reply:
This is actually related to mutual followers feature but I added both of them under one pull request